Author: Stefan Fritsch <sf@debian.org>
Description: Fix generation of docs for out-of-tree builds and newer doxygen versions
Index: apr-1.4.5/docs/doxygen.conf
===================================================================
--- apr-1.4.5.orig/docs/doxygen.conf	2005-02-08 02:12:15.000000000 +0300
+++ apr-1.4.5/docs/doxygen.conf	2011-05-24 16:19:06.021240174 +0400
@@ -1,6 +1,6 @@
 PROJECT_NAME="Apache Portable Runtime"
 
-INPUT=.
+INPUT=../include include
 QUIET=YES
 RECURSIVE=YES
 FILE_PATTERNS=*.h
Index: apr-1.4.5/include/apr.h.in
===================================================================
--- apr-1.4.5.orig/include/apr.h.in	2011-05-24 16:18:59.971241092 +0400
+++ apr-1.4.5/include/apr.h.in	2011-05-24 16:19:06.038579344 +0400
@@ -445,11 +445,11 @@
 /** 
  * Thread callbacks from APR functions must be declared with APR_THREAD_FUNC, 
  * so that they follow the platform's calling convention.
- * <PRE>
- *
+ */
+/**
+ * @code
  * void* APR_THREAD_FUNC my_thread_entry_fn(apr_thread_t *thd, void *data);
- *
- * </PRE>
+ * @endcode
  */
 #define APR_THREAD_FUNC       @apr_thread_func@
 
@@ -481,11 +481,11 @@
  * APR_DECLARE_NONSTD(), as they must follow the C language calling convention.
  * @see APR_DECLARE @see APR_DECLARE_DATA
  * @remark Both the declaration and implementations must use the same macro.
- * <PRE>
- *
+ */
+/**
+ * @code
  * APR_DECLARE_NONSTD(rettype) apr_func(args, ...);
- *
- * </PRE>
+ * @endcode
  */
 #define APR_DECLARE_NONSTD(type)     type
 
@@ -496,12 +496,12 @@
  * @remark Note that the declaration and implementations use different forms,
  * but both must include the macro.
  * 
- * <PRE>
- *
+ */
+/**
+ * @code
  * extern APR_DECLARE_DATA type apr_variable;\n
  * APR_DECLARE_DATA type apr_variable = value;
- *
- * </PRE>
+ * @endcode
  */
 #define APR_DECLARE_DATA
 
